.github/workflows/python-ci.yml

```
@@ -70,3 +70,26 @@ jobs:
         docker run --rm -i hadolint/hadolint < Dockerfiles/sync_firehose_stream.Dockerfile
         docker run --rm -i hadolint/hadolint < Dockerfiles/sync_most_liked_feed.Dockerfile
         docker run --rm -i hadolint/hadolint < Dockerfiles/ml_inference_perspective_api.Dockerfile
+
+  frontend-lint:
+    runs-on: ubuntu-latest
+    if: contains(github.event.head_commit.modified, 'bluesky_database/frontend/') || contains(github.event.head_commit.added, 'bluesky_database/frontend/') || contains(github.event.head_commit.removed, 'bluesky_database/frontend/')
+    defaults:
+      run:
+        working-directory: bluesky_database/frontend
+    steps:
+    - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)